mne/preprocessing/maxwell.py

```diff
@@ -321,6 +321,10 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
         sss_ctc = _read_ctc(cross_talk)
         ctc_chs = sss_ctc['proj_items_chs']
         meg_ch_names = [info['ch_names'][p] for p in meg_picks]
+        # checking for extra space ambiguity in channel names
+        # between old and new fif files
+        if meg_ch_names[0][3] != ctc_chs[0][3]:
+            ctc_chs = _clean_names(ctc_chs, remove_whitespace=True)
```

Comment:
Actually, I tried it but test were unsuccessful, for `meg_ch_names` this needs to be implemented much lower as following throws error

```
ctc_picks = pick_channels(ctc_chs,
                                  [info['ch_names'][c]
                                   for c in meg_picks[good_picks]])
        assert len(ctc_picks) == len(good_picks)  # otherwise we errored
```